packages/tempus/unit_test/Tempus_UnitTest_EDIRK_TrapezoidalRule.cpp

```diff
@@ -64,4 +65,163 @@ TEUCHOS_UNIT_TEST(EDIRK_TrapezoidalRule, AppAction)
 }
 
 
+// ************************************************************
+// ************************************************************
+
+class StepperRKModifierEDIRK_TrapezoidaTest
```

Comment:
There seems to be several places where "Trapezoida" is used.  Should it be "Trapezoidal"?